components/src/TruncatedText/TruncatedText.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { Tooltip } from "../Tooltip";
+import { Text } from "../Type";
+
+const StyledTruncatedText = styled(Text)(({ hoverable }) => ({
```

Comment:
i now realize this was missing from the acceptance criteria, but the text that gets truncated is sometimes (and often) a title, so there should be a way to pass something other than Text. i don't think it needs to be generic, just the ability for it to at least be Text, Title or SectionTitle... maybe by passing the `as` prop down to here and then documenting the possibility?